content/docs/reference-dom-elements.md

```diff
@@ -88,6 +88,18 @@ function ComponentWithTransition() {
 
 Style keys are camelCased in order to be consistent with accessing the properties on DOM nodes from JS (e.g. `node.style.backgroundImage`). Vendor prefixes [other than `ms`](http://www.andismith.com/blog/2012/02/modernizr-prefixed/) should begin with a capital letter. This is why `WebkitTransition` has an uppercase "W".
 
+When specifying a pixel value for your inline style prop, React automatically appends the string "px" for you after your number value, so this works:
+
+```js
+const divStyle = {
+  height: 10  // rendered as "height:10px"
+};
+
+function HelloWorldComponent() {
+  return <div style={divStyle}>Hello World!</div>;
+}
+```
+
```

Comment:
I think the [section about which props are unitless](https://github.com/facebook/react/blob/8cac523beaaacfeae179ca14a1d8a46d82892016/docs/tips/06-style-props-value-px.md) is also worth adding just to avoid potential confusion that _all_ numeric style props are stringified like this.

Reply:
Should it be updated to reflect the latest changes in [this file](https://github.com/facebook/react/blob/4131af3e4bf52f3a003537ec95a1655147c81270/src/renderers/dom/shared/CSSProperty.js#L15-L59)(20 of them are not on the list)?

Reply:
Sure, or maybe just mention the _type_ of props that aren't converted and list a few examples?

Reply:
Alright, I left it as is and include a link to the file.